ios/RCTMapboxGL/RCTMapboxGL.m

```diff
@@ -422,8 +434,12 @@ - (void)setVisibleCoordinateBounds:(MGLCoordinateBounds)bounds edgePadding:(UIEd
 - (void)selectAnnotation:(NSString*)selectedId animated:(BOOL)animated;
 {
     RCTMGLAnnotation * annotation = [_annotations objectForKey:selectedId];
-    if (!annotation) { return; }
-    [_map selectAnnotation:annotation animated:animated];
+    if (!annotation) {
+        _selectedAnnotation = nil;
+        return; 
+        }
```

Comment:
4 extra spaces of indentation here. Brackets should close on the same level as the if